osgi/src/main/java/org/pentaho/ctools/cda/DataFactoryRegister.java

```diff
@@ -0,0 +1,36 @@
+package org.pentaho.ctools.cda;
+
+import org.pentaho.reporting.engine.classic.core.metadata.DataFactoryRegistry;
+import org.pentaho.reporting.engine.classic.core.metadata.DefaultDataFactoryCore;
+import org.pentaho.reporting.engine.classic.core.metadata.DefaultDataFactoryMetaData;
+import org.pentaho.reporting.engine.classic.core.wizard.DefaultDataAttributeCache;
+import pt.webdetails.cda.CdaEngine;
+import pt.webdetails.cda.InitializationException;
+import pt.webdetails.cda.utils.streaming.SQLStreamingReportDataFactory;
+
+public class DataFactoryRegister {
+  public DataFactoryRegister( CdaEngine cdaEngine ) {
+  }
+
+  public void init() throws InitializationException {
+    registerCustomDataFactories();
+
+    // This is just to force the import of org.pentaho.reporting.engine.classic.core.wizard
+    // TODO Figure out a better solution or simply add the Import-Package instruction to the maven-bundle-plugin configuration
+    DefaultDataAttributeCache.class.getName();
```

Comment:
Figure out a better solution or simply add the Import-Package instruction to the maven-bundle-plugin configuration.